src/images/svg/trade-types/cryptocurrencies.svg

```diff
@@ -0,0 +1,13 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 0 48 48">
```

Comment:
Hi @fiona-deriv  can we compress this SVG?

Reply:
yeah sure